.eslintrc.json

```diff
@@ -12,14 +12,40 @@
     "Promise": false
   },
   "rules": {
+    "array-bracket-spacing": "error",
```

Comment:
Do you think that we should instead use `  "extends": "google",`?
See https://github.com/google/eslint-config-google

Reply:
That's what I based off, the main changes (apart from `max-len` mentioned) make life easier for now:
* No ES6 rules confusing things.
* No `require-jsdoc` which is overkill for the non-library files.
* No `one-var` which is confusing without `const` and `let` being enforced.

Those could be overridden back in line, which might be clearer?